src/Microsoft.AspNet.Mvc.Core/Formatters/MediaTypeSegmentWithQuality.cs

```diff
@@ -9,7 +8,6 @@ namespace Microsoft.AspNet.Mvc
     /// <summary>
     /// Represents a media type with its associated quality.
     /// </summary>
-    [DebuggerDisplay("{MediaType}")]
```

Comment:
This is just left over from breaking the PR, It will go away when I join all three parts and squash them